### ISSUE TITLE
Are these sympy benchmarks useful?

### ISSUE THREAD

**Issue by moorepants** [bjodah/sympy_benchmarks_bjodah#7]
Maybe we should merge these into this code:

https://github.com/sympy/sympy/tree/master/sympy/benchmarks

**Comment by bjodah** [comment]
Agreed, let's just give them a little more descriptive names perhaps?
